test/proper_tests.erl

```diff
@@ -1366,8 +1366,11 @@ sampleshrink_test_() ->
 %%------------------------------------------------------------------------------
 
 examples_are_ok_test_() ->
-    [{timeout, 42, ?_assertEqual([], proper:module(M))}
-     || M <- [b64,elevator_fsm,ets_statem,mastermind,pdict_statem,stack]].
+    %% No need to test more in order to check that the examples work.
```

Comment:
This change is not OK.  I suggest we keep running as many tests as we can with the default `numtests` and we only run the two car tests with 42.